doc/classes/ProjectSettings.xml

```diff
@@ -269,6 +269,13 @@
 		<member name="application/run/disable_stdout" type="bool" setter="" getter="" default="false">
 			If [code]true[/code], disables printing to standard output in an exported build.
 		</member>
+		<member name="application/run/flush_stdout_on_print" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], flushes the standard output stream every time a line is printed. This affects both terminal logging and file logging.
+			When running a project, this setting must be enabled if you want logs to be collected by service managers such as systemd/journalctl. This setting is disabled by default on release builds, since flushing on every printed line will negatively affect performance if lots of lines are printed in a rapid succession. Also, if this setting is enabled, logged files will still be written successfully if the application crashes or is otherwise killed by the user (without being closed "normally").
+			[b]Note:[/b] Regardless of this setting, the standard error stream ([code]stderr[/code]) is always flushed when a line is printed to it.
+		</member>
+		<member name="application/run/flush_stdout_on_print.debug" type="bool" setter="" getter="" default="true">
```

Comment:
Needs a description too.

Reply:
Do we have a description for other settings overridden for specific feature tags? I haven't seen any.

Reply:
Yes, check all the `.mobile` or `.web` overrides:
```
		<member name="audio/output_latency" type="int" setter="" getter="" default="15">
			Output latency in milliseconds for audio. Lower values will result in lower audio latency at the cost of increased CPU usage. Low values may result in audible cracking on slower hardware.
		</member>
		<member name="audio/output_latency.web" type="int" setter="" getter="" default="50">
			Safer override for [member audio/output_latency] in the Web platform, to avoid audio issues especially on mobile devices.
		</member>

		<member name="rendering/quality/glow/upscale_mode" type="int" setter="" getter="" default="1">
			Sets how the glow effect is upscaled before being copied onto the screen. Linear is faster, but looks blocky. Bicubic is slower but looks smooth.
		</member>
		<member name="rendering/quality/glow/upscale_mode.mobile" type="int" setter="" getter="" default="0">
			Lower-end override for [member rendering/quality/glow/upscale_mode] on mobile devices, due to performance concerns or driver support.
		</member>
```

I think it's important to show feature/platform overrides in the ProjectSettings, and explain why the default value is different.